ext/http/00_serve.ts

```diff
@@ -156,9 +158,15 @@ class InnerRequest {
         );
       }
     }
+    if (!signalAbortError) {
+      signalAbortError = new DOMException(
+        "The signal has already been aborted.",
```

Comment:
While we're at it, we could make this a nicer error - like either "The request has been completed." or "The request has been cancelled."